### ISSUE TITLE
Manual pages: Please don't call options "flags" 

### ISSUE THREAD

**Issue by msiism** [restic/restic#3359]
Currently, the SYNOPSIS sections of all of _restic_'s manual pages call options "flags". They should say "options" instead.

Rationale:

The standard name for the section describing a command's options in manual pages is OPTIONS, and restic already follows that standard. So, saying "flags" instead in the SYNOPSIS (or anywhere) is potentially confusing.

While "flags" has been the term for command options historically, "options" is a better term because it indicates way more clearly what these arguments are.

**Comment by rawtaz** [comment]
You are right that there are some inconsistencies. Throughout PRs and so on we've tried to keep it to "option", but it's still a bit mixed. Unfortunately the choice is further complicated by the fact that the library we use for parsing these options, as well as the corresponding library in Go itself, actually calls them [flags](https://github.com/spf13/cobra#flags)..

**Comment by msiism** [comment]
Right, I've already been told on IRC that this very much a library issue. But I was asked to report it as a _restic_ bug anyway, for starters.
